tsconfig.json

```diff
@@ -1,18 +1,13 @@
 {
-  "extends": "./node_modules/gts/tsconfig-google.json",
+  "extends": "./tsconfig.full.json",
```

Comment:
is tsconfig.full.json different from tsconfig-google.json? If so, what's the rationale here?

Reply:
Edited abstract above. Hope it makes sense